### PR TITLE
Adding integratouch.json and symphony.json to source

### PR DESCRIPTION
**Pull request by frogythegreat**
Resubmitting PR with corrected authorship, 
Adding integratouch.json and symphony.json to source

### CODE REVIEW

**Comment by frogythegreat** [comment]
I believe I have verified all of the above, I am committing and submitting the PR as dmccheyne@integratouch.com, the account is in the google group assigned when we signed the company CLA. Can you possibly provide any more information about where that verification is failing so I can follow up? Thanks!

**Comment by frogythegreat** [comment]
I signed it!

File: sources/integratouch.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (pfrisella) [inline sources/integratouch.json]: Org entries should go under the organizations path

Comment (frogythegreat) [reply sources/integratouch.json]: We've moved this file. Thanks!

**Comment by pfrisella** [comment]
LGTM, thanks for adding!

**Pull request merged**